src/jdbc/Main.java

```diff
@@ -0,0 +1,43 @@
+package src.jdbc;
+
+import java.sql.*;
+
+public class Main {
+    public static void main(String[] args) throws SQLException {
+        Connection connection = ConnectionManager.getConnection();
+        try {
+            Statement statement = connection.createStatement();
+        } catch (SQLException e) {
+            throw new RuntimeException(e);
+        }
+        String query = """
+                insert into contact
+                (contact, contact_type)
+                VALUES (?, ?)
+                """;
+
+        String allContact = """
+               select * from contact;
+               """;
+        addNewContact(connection, query,"Hide","Hide");
+        addNewContact(connection, query,"Java","Java");
+        selectAllContact(connection, allContact);
+    }
+
+    private static void addNewContact(Connection connection, String query, String contact, String contact_type) throws SQLException {
```

Comment:
Minor note, due to java naming convention, we need to use camel case style(camelCase) for variable names.

you have contact_type, it should be contactType